tests_new/integration_tests/aws_clients/iam.py

```diff
@@ -34,7 +43,9 @@ def create_role(self, account_id, role_name):
                 {{
```

Comment:
nit: I'd make this a dict and then do a `json.dumps`

Reply:
done